### DISCUSSION TITLE
info server bungee

### DISCUSSION THREAD

**Discussion by pinuxio** [Zrips/CMI#4553] [Q&A]
**Question: **
Hello, being the admin and founder of the server, is there a way to control all the users of the various servers? example:
I am on the vanilla server but I would also like to check users in towny and creative

**Comment by EkonC** [comment]
What do you mean by controling them or checking them? If you want to list connected players, use command /glist, to ban them, use a punish system that works on bungee (like AdvancedBan or LiteBans). Everything other - use console or simply connect to that server.

**Comment by CloudeLecaw** [comment]
May he thought there will be a gui or something to watch over all or use commands over all. But this wichlist does not exist. You said u have a vanilla server?..how could this be bungee ;)
But its ok, we know what You mean.
Like in the post beliw said allready you can use the bingeeonly commans and also snyc Economy or Chats or Portas but you will need to tp to their serverpart to be able to controll something
